src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/comparator/Car_Transmission_Levenshtein_Comparator.java

```diff
@@ -0,0 +1,51 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.comparator;
+
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.model.Car;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.Comparator;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.ComparatorLogger;
+import de.uni_mannheim.informatik.dws.winter.model.Correspondence;
+import de.uni_mannheim.informatik.dws.winter.model.Matchable;
+import de.uni_mannheim.informatik.dws.winter.model.defaultmodel.Attribute;
+import de.uni_mannheim.informatik.dws.winter.similarity.string.LevenshteinSimilarity;
+
+public class Car_Transmission_Levenshtein_Comparator implements Comparator<Car, Attribute> {
+    private static final long serialVersionUID = 1L;
+
+    private LevenshteinSimilarity sim = new LevenshteinSimilarity();
+
+    private ComparatorLogger comparisonLog;
+
+
+    @Override
+    public double compare(Car record1,
+                          Car record2,
+                          Correspondence<Attribute, Matchable> schemaCorrespondence) {
+        String transmission1 = record1.getTransmission().toLowerCase();
```

Comment:
s.o.
And I think a Levenshtein Transmission comparator is already checked in. Probably you could just extend the existing file with your preprocessing?